src/analysis/simulation.jl

```diff
@@ -98,7 +98,7 @@ function simulate(data::Dict{String,Any}, solution::Dict{String,Any},
         diameter = wn.nodes._data[tank_name].diameter
 
         # enforce the initial levels of tanks 
-        wn.nodes._data[tank_name].init_level = solution["1"]["tank"][tank_index_dict[tank_name]]["V"]/(1/4*pi*diameter^2)
+        wn.nodes._data[tank_name].init_level = wm_solution["solution"]["nw"]["1"]["tank"][tank_index_dict[tank_name]]["V"]/(1/4*pi*diameter^2)
```

Comment:
For safety, this should be computed as the head minus the elevation at the tank node instead of an operation on the volume. I'd advise using the pressure component of the solution, "p," which gives the same thing.

Reply:
Not sure why computing something from the volume shouldn't work consistently, but this has been changed now. Although I am making an update to use the correct node.

Reply:
I noticed that using the tank node's pressure will give the wrong (impossible) tank level sometimes - for example, the pressure of a tank node solved by WaterModels could be negative. However, the pressure of the artificial node added next to the tank seems to be always consistent with the real tank level that is calculated from the tank volume. Therefore, using 'pressure' to compute the tank level actually now involves more codes as I need to first identify the 'artificial node' attached to the tank. 

I do think using volume is probably more straightforward.

Reply:
I think this was when the artificial node and tank were reversed. I am about to make a commit that uses the pressure for the tank-node. No negative pressure and levels.

Reply:
It is fine to use volume for now since we assume only cylindrical tanks, but this could change in the future if we decide to model non-cylindrical tanks.